cypress/Dockerfile

```diff
@@ -44,4 +44,7 @@ RUN useradd -ms /bin/bash hmda_cypress_user && \
 
 USER hmda_cypress_user
 
-RUN yarn cypress install
\ No newline at end of file
+RUN yarn cypress install
+
+# Copy load-test file from S3 if not already present
```

Comment:
My original plan was to perform this copy [inside of the running pod](https://github.com/cfpb/hmda-frontend/pull/1330/files#diff-d92cde168d4ccc1a2eff895bf45b8573cd02f7c0ec4580c62f58edb4c999a589R55), however I was unable to set the correct file permissions on `s3://cfpb-hmda-public/cypress/2020-FRONTENDTESTBANK9999-MAX.txt` to make it publicly accessible via `curl`, hence this approach.  
 
[Access Denied](https://s3.amazonaws.com/cfpb-hmda-public/cypress/2020-FRONTENDTESTBANK9999-MAX.txt); only downloadable with `aws` cli, using the`svc_hmda` profile.

Reply:
did you create this file through the aws web console? If this file should be / could be made public, you can do so in the console, there's a `Make public using ACL` selection you can follow.